src/assets/Enums.ts

```diff
@@ -0,0 +1,10 @@
+export const PROHIB_CLEARANCE_TEST_TYPE_IDS = {
+  IDS: [
```

Comment:
Looks like there are few test type ids are missing, 11, 68 and 81 are all prohibitions types in the taxonomy

Reply:
These additional test types id's have now been added in